policy/modules/contrib/targetd.te

```diff
@@ -83,27 +96,83 @@ storage_raw_read_removable_device(targetd_t)
 sysnet_read_config(targetd_t)
 
 optional_policy(`
-    gnome_read_generic_data_home_dirs(targetd_t)
+	gnome_read_generic_data_home_dirs(targetd_t)
 ')
 
 optional_policy(`
-    lvm_domtrans(targetd_t)
+	lvm_domtrans(targetd_t)
 ')
 
 optional_policy(`
-    modutils_read_module_config(targetd_t)
+	modutils_read_module_config(targetd_t)
 ')
 
 optional_policy(`
-    rpc_manage_nfs_state_data(targetd_t)
+	rpc_manage_nfs_state_data(targetd_t)
 ')
 
 optional_policy(`
-    rpm_dontaudit_read_db(targetd_t)
-    rpm_dontaudit_exec(targetd_t)
+	rpm_dontaudit_read_db(targetd_t)
+	rpm_dontaudit_exec(targetd_t)
 ')
 
 optional_policy(`
-   udev_read_pid_files(targetd_t)
+	udev_read_pid_files(targetd_t)
 ')
 
+########################################
+#
+# targetclid local policy
+#
+allow targetclid_t self:capability dac_override;
+allow targetclid_t self:fifo_file rw_fifo_file_perms;
+allow targetclid_t self:system module_load;
+allow targetclid_t self:unix_stream_socket create_stream_socket_perms;
+
+manage_dirs_pattern(targetclid_t, targetclid_home_t, targetclid_home_t)
+manage_files_pattern(targetclid_t, targetclid_home_t, targetclid_home_t)
+userdom_admin_home_dir_filetrans(targetclid_t, targetclid_home_t, dir, ".targetcli")
+
+manage_dirs_pattern(targetclid_t, targetclid_var_run_t, targetclid_var_run_t)
```

Comment:
Is manage_dirs required? I see only a plain file and sock_file in the fc file.

Reply:
you are right, fixed, thank you